pkg/plugins/backendplugin/pluginextensionv2/generate.sh

```diff
@@ -13,4 +13,4 @@ DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 
 cd "$DIR"
 
-protoc -I ./ *.proto --go_out=plugins=grpc:./
+protoc -I ./ *.proto --go_out=. --go-grpc_out=.
```

Comment:
I couldn't re-generate protobuf files without this change.

This is also the same command we use to generate the protobuf files in the SDK:

https://github.com/grafana/grafana-plugin-sdk-go/blob/6ecd6938cfed8cbfe1cdd1a43696321e23c362d1/proto/generate.sh#L16-L19